core/src/main/java/lucee/runtime/ComponentImpl.java

```diff
@@ -1596,6 +1596,8 @@ private static void metaUDFs(PageContext pc, ComponentImpl comp, Struct sct, int
 		if (arr.size() != 0) {
 			Collections.sort(arr, new ComparatorImpl());
 			sct.set(KeyConstants._functions, arr);
+		}else{
```

Comment:
```suggestion
		} else {
```

Reply:
@jbampton manual format changes are not necessary, we auto format all code
https://github.com/lucee/Lucee/tree/5.3/core/.settings